apix-impl/50/src/main/java/eu/xenit/apix/alfresco50/SpringConfiguration.java

```diff
@@ -1,9 +0,0 @@
-package eu.xenit.apix.alfresco50;
```

Comment:
Weren't these required to make the workflow engine configurable? (I seem to remember that config was only loaded if this file was present? @rschev do you remember whether this is correct?)

Reply:
Good question. I tested this change by running all integration test on all versions -> everything succeeded.
Do you think the workflow engine is not covered by those?

Reply:
I checked the tests and it looks like the default engine is sufficiently covert. The APS option is not, but it is as a whole unfinished and not a priority. Additionally, the comment in the `ApixSpringConfiguration` class makes me think we do not need it atm.